compiler-core/src/package_interface.rs

```diff
@@ -372,11 +377,67 @@ impl PackageInterface {
                 .iter()
                 .filter(|module| !package.config.is_internal_module(module.name.as_str()))
                 .map(|module| (module.name.clone(), ModuleInterface::from_module(module)))
+                .chain(
+                    package
+                        .cached_modules
+                        .iter()
+                        .filter(|module| !package.config.is_internal_module(module.name.as_str()))
+                        .map(|interface| {
+                            (
+                                interface.name.clone(),
+                                ModuleInterface::from_untyped_module(interface),
```

Comment:
Let's always use the interface and then delete the code for doing it from the AST.

Reply:
So I delete the current `ModuleInterface::from_module` and rename the `ModuleInterface::from_untyped_module` to `ModuleInterface::from_interface`?